core/protos/protos/network.proto

```diff
@@ -65,6 +65,7 @@ message Message {
     message BlockResponse {
         uint64 request_id = 1;
         repeated CoupledBlock response = 2;
+        uint64 best_index = 3;
```

Comment:
Please comment in the proto itself and in the corresponding Rust struct what best index corresponds to.